packages/core/parcel/package.json

```diff
@@ -14,9 +14,12 @@
     "type": "git",
     "url": "https://github.com/parcel-bundler/parcel.git"
   },
-  "bin": "src/bin.js",
+  "bin": "bin/parcel.js",
   "main": "lib/bin.js",
   "source": "src/bin.js",
+  "scripts": {
+    "dev:prepare": "rm -rf ./lib/ && mkdir -p lib && cp ./bin/dev-bin.js ./lib/bin.js"
```

Comment:
/nit Although this works in CI because Actions has these global commands available, it wouldn't work in windows. That's why most of the scripts use `rimraf` and you'd need equivalents for the others too. I'm not sure if windows is well supported for contributing in general though.